geoportal/c2cgeoportal_geoportal/scripts/theme2fts.py

```diff
@@ -289,3 +289,53 @@ def _add_layer(
             self._add_fts(layer, interface, "add_layer", role)
 
         return fill
+
+    def _render_label(
+        self,
+        item: "c2cgeoportal_commons.models.main.TreeItem",
+        lang: str,
+    ) -> Optional[str]:
+        patterns = item.get_metadata("searchLabelPattern")
+        if not patterns:
+            return None
```

Comment:
I think that's more coherent
```suggestion
            return self._[lang].gettext(item.name)
```